src/test/java/com/google/alpollo/AnalysisTest.java

```diff
@@ -166,8 +168,9 @@ public void checkEntitiesWithMetadata() throws IOException {
     String responseString = responseWriter.toString();
 
     List<SongEntity> actual = gson.fromJson(responseString, new TypeToken<List<SongEntity>>(){}.getType());
-    List<SongEntity> expected = Arrays.asList(new SongEntity("Google", 0.88, "ORGANIZATION", WIKI_LINK_GOOGLE),
-        new SongEntity("results", 0.12, "OTHER", EMPTY_STRING));
+    List<SongEntity> expected = Arrays.asList(
+        new SongEntity("Google", 0.88, new HashSet<String>(Arrays.asList("ORGANIZATION")), WIKI_LINK_GOOGLE),
+        new SongEntity("results", 0.12, new HashSet<String>(Arrays.asList("OTHER")), EMPTY_STRING));
 
       Assert.assertThat(actual, CoreMatchers.is(expected));
   }
```

Comment:
Let's add another test where the API returns the same type for the same entity, as I saw an occurrence of that happening in production, i.e.:

```
{ "mountain", "OTHER", 0.04 }
{ "mountain", "OTHER", 0.03 }
```

I'd expect to combine this results